pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -45,7 +45,7 @@ type OperatorVersionSpec struct {
 	Dependencies []OperatorDependency `json:"dependencies,omitempty"`
 
 	// UpgradableFrom lists all OperatorVersions that can upgrade to this OperatorVersion.
-	UpgradableFrom []OperatorVersion `json:"upgradableFrom,omitempty"`
+	UpgradableFrom []corev1.ObjectReference `json:"upgradableFrom,omitempty"`
```

Comment:
🤷‍♀ we don't really use this field, so I wouldn't be against removing it until we know what to do with it :)